### PR TITLE
SDK 0.0.18 update

### LINKED ISSUE #372: Update to TasitSDK 0.0.18 and use the new API throughout
The latest version of `tasit-sdk` has some break changes:
- `action.waitForNonceToUpdate()` is now `action.waitForOneConfirmation()`;
- `action` now is sent using `send()` function.
- `Contract.wallet` is now `Contract.account` (getWallet, setWallet, removeWallet functions).

### LINKED ISSUE #380: Fix npm install test failure
```
npm ERR! path /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view
npm ERR! code EISGIT
npm ERR! git /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view: Appears to be a git repo or submodule.
npm ERR! git     /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

### PR DESCRIPTION
**Pull request by pcowgill**
### Issue link

https://github.com/tasitlabs/tasit/issues/372

### Auto-close the issue?

Closes #372
Closes #376 
Closes #380

### Types of changes

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### Notes

This issue is still TODO https://github.com/tasitlabs/tasit/issues/151

### CODE REVIEW

File: decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js [hunk 1]
```diff
@@ -53,36 +53,44 @@ export class EthereumSignUpScreen extends React.Component {
 
       const fundWithEthers = async accountAddress => {
         const action = fundAccountWithEthers(accountAddress);
+        action.send();
+        console.info("Sent the ETH transfer action");
         const actionId = await action.getId();
+        console.info({ actionId });
```
Comment (pcowgill) [inline decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js]: @marcelomorgado I'm seeing

```
[00:37:26] Built the action to transfer ETH to ephemeral account

[00:37:26] Sent the ETH transfer action
```

but not the actionId log

**Comment by pcowgill** [comment]
As for this `npm install` error in CI, I got it locally and removed the package before doing another npm install.

```
npm ERR! path /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view
npm ERR! code EISGIT
npm ERR! git /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view: Appears to be a git repo or submodule.
npm ERR! git     /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

Maybe I need to redo it and make sure it's reflected in `package-lock.json`

**Comment by pcowgill** [comment]
> As for this `npm install` error in CI, I got it locally and removed the package before doing another npm install.
> 
> ```
> npm ERR! path /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view
> npm ERR! code EISGIT
> npm ERR! git /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view: Appears to be a git repo or submodule.
> npm ERR! git     /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view
> npm ERR! git Refusing to remove it. Update manually,
> npm ERR! git or move it out of the way first.
> ```
> 
> Maybe I need to redo it and make sure it's reflected in `package-lock.json`

Here's an issue for this - it's an issue in the develop branch too https://github.com/tasitlabs/tasit/issues/380

**Comment by marcelomorgado** [comment]
The `action.send()` function should be called with `await` if used the `waiting` style.

**Comment by marcelomorgado** [comment]
CI test passed after disabling cache (same issue occured on TasitSDK)

**Comment by pcowgill** [comment]
> The `action.send()` function should be called with `await` if used the `waiting` style.

Ah ok, that’s good to know. Thanks

**Comment by pcowgill** [comment]
> CI test passed after disabling cache (the same issue occurred on TasitSDK)
> I've changed the PR description adding the reference to the issue.

Oh okay, thanks for fixing that!

**Pull request merged**